### PR TITLE
Add Orange3-Geo

### PR DESCRIPTION
**Pull request by kernc**
cc: @astaric

### CODE REVIEW

File: recipes/orange3-geo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - python
+    - orange3
+    - scikit-learn
+    - pandas
+    - scipy >=0.17
+    - shapely
+
+test:
+  imports:
+    - orangecontrib.geo
```
Comment (kernc) [inline recipes/orange3-geo/meta.yaml]: Orange3-Geo defines [some tests in _setup.py_](https://github.com/biolab/orange3-geo/blob/0371f967bf4059cdae0146c709843045e8d6282e/setup.py#L87), but I don't know how to access it.

File: recipes/orange3-geo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  home: http://github.com/biolab/orange3-geo
+  license: GPL-3.0
+  license_family: GPL
+  summary: 'Orange add-on for dealing with geography and geo-location'
+  dev_url: http://github.com/biolab/orange3-geo
+
+extra:
+  recipe-maintainers:
+    - kernc
+    - astaric
```
Comment (jakirkham) [inline recipes/orange3-geo/meta.yaml]: Are you ok being a maintainer here @astaric?

File: recipes/orange3-geo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - simplejson
+
+test:
+  imports:
+    - orangecontrib.geo
+
+about:
+  home: http://github.com/biolab/orange3-geo
+  license: GPL-3.0
+  license_family: GPL
```
Comment (jakirkham) [inline recipes/orange3-geo/meta.yaml]: Please ask upstream to add a license file and include it in the `MANIFEST.in` to allow packaging it in the future. Also please add a comment with `license_file: <license filename>` where `<license filename>` is replaced with the license file name as added in the repo and potentially location if it is not in the top directory of the repo.

**Comment by jakirkham** [comment]
Mostly good. Note about the license file and a check on adding a maintainer.

File: recipes/orange3-geo/meta.yaml (hunk 2 above)
Comment (astaric) [inline recipes/orange3-geo/meta.yaml]: I ok with it

File: recipes/orange3-geo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - scipy >=0.17
+    - shapely
+    - simplejson
+
+test:
+  imports:
+    - orangecontrib.geo
+
+about:
+  home: http://github.com/biolab/orange3-geo
```
Comment (jakirkham) [inline recipes/orange3-geo/meta.yaml]: `https`?

File: recipes/orange3-geo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  license: GPL-3.0
+  license_family: GPL
+  summary: 'Orange add-on for dealing with geography and geo-location'
+  dev_url: http://github.com/biolab/orange3-geo
```
Comment (jakirkham) [inline recipes/orange3-geo/meta.yaml]: `https`?

**Pull request merged**